webpack.common.js

```diff
@@ -20,9 +20,9 @@ module.exports = {
     ],
     vendor: ['phaser']
   },
-  context: path.resolve(__dirname, 'src'),
+  context: path.resolve(paths.src),
```

Comment:
The `path.resolve()` function call is unnecessary since we resolved the paths earlier. This is my bad.